frontends/bmd/src/pages/admin_screen.tsx

```diff
@@ -57,12 +59,9 @@ export function AdminScreen({
     const precinctId = event.currentTarget.value;
 
     if (precinctId === ALL_PRECINCTS_OPTION_VALUE) {
-      updateAppPrecinct({ kind: PrecinctSelectionKind.AllPrecincts });
+      updateAppPrecinct(ALL_PRECINCTS_SELECTION);
     } else if (precinctId) {
-      updateAppPrecinct({
-        kind: PrecinctSelectionKind.SinglePrecinct,
-        precinctId,
-      });
+      updateAppPrecinct(getSinglePrecinctSelection(precinctId));
```

Comment:
I don't feel strongly about this! `constructSinglePrecinctSelection` might be a clearer name than `getSinglePrecinctSelection`, which reads as if we're retrieving something from a store

Reply:
Renamed to `singlePrecinctSelectionFor`.